src/hb-common.h

```diff
@@ -47,14 +47,10 @@
 # endif /* !__cplusplus */
 #endif
 
-#if defined (_SVR4) || defined (SVR4) || defined (__OpenBSD__) || \
-    defined (_sgi) || defined (__sun) || defined (sun) || \
-    defined (__digital__) || defined (__HP_cc)
-#  include <inttypes.h>
-#elif defined (_AIX)
+#if defined (_AIX)
 #  include <sys/inttypes.h>
 #elif defined (_MSC_VER) && _MSC_VER < 1600
-/* VS 2010 (_MSC_VER 1600) has stdint.h */
+/* VS 2010 (_MSC_VER 1600) has inttypes.h */
```

Comment:
VS2013 has inttypes.h, not 2010: You should add _MSC_VER 1800 special casings from the previous PR to avoid any possible clashes

Reply:
Fixed. Thanks.

Reply:
Hm, I hope a stdint.h from _MSC_VER < 1800 won't cause any clashes..

Reply:
In any case, I'm OK with this version of the patch to solve #4619